src/textures/Texture.js

```diff
@@ -110,6 +111,7 @@ Texture.prototype = Object.assign( Object.create( EventDispatcher.prototype ), {
 		this.anisotropy = source.anisotropy;
 
 		this.format = source.format;
+		this.internalFormat = source.internalFormat;
```

Comment:
```js
this.gpuFormat = source.gpuFormat;
```

Reply:
How about calling it `.internalFormat` consistently, and crossing the WebGPU bridge when we get to it?

Reply:
I thought `.gpuFormat` was a good name.
`.internalFormat` can be confusing. Is it internal in Three.js? Or in WebGL?

Reply:
It is the internal format as defined in the webGL spec. Personally, I would adhere to that nomenclature.

Reply:
I fixed that anyway in the last commit.

`internalFormat` also makes more sense to me, but `gpuFormat` is acceptable I guess. Let me know which one you want.

Reply:
Lets go with `internalFormat` then.